File: backend/src/main/java/com/coing/domain/coin/orderbook/entity/OrderbookUnit.java
package com.coing.domain.coin.orderbook.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;

@Getter
@AllArgsConstructor
@Builder
public class OrderbookUnit {
	private Double askPrice; // 매도 호가
	private Double bidPrice; // 매수 호가
	private Double askSize;  // 매도 잔량
	private Double bidSize;  // 매수 잔량
}

File: backend/src/main/java/com/coing/domain/coin/orderbook/entity/OrderbookUnit.kt
package com.coing.domain.coin.orderbook.entity

data class OrderbookUnit(
    val askPrice: Double, // 매도 호가
    val bidPrice: Double, // 매수 호가
    val askSize: Double, // 매도 잔량
    val bidSize: Double, // 매수 잔량
)